metaflow/plugins/cards/ui/src/App.svelte

```diff
@@ -12,7 +13,7 @@
   import Nav from "./components/aside-nav.svelte";
 </script>
 
-<div class="container">
+<div class="container card">
```

Comment:
do we risk `card` being too generic if we don't control the whole page?  Its a pretty common UI term.  I wonder if instead we use an ID to to establish precedence over any classes?  ie.  find/replace `.card` with `#app-card`

Reply:
I agree we should change the name to something less generic - I like `mf-card`. I considered an ID but thought there may be multiple of these cards on a page. But your point about increased specificity may override that

Reply:
good point, lets go with the more specific class then.